<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VideoFrame</class>
 <widget class="QFrame" name="VideoFrame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1080</width>
    <height>621</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <property name="leftMargin">
    <number>40</number>
   </property>
   <item>
    <widget class="QFrame" name="listFrame">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QFrame" name="frame">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <property name="leftMargin">
              <number>3</number>
             </property>
             <property name="topMargin">
              <number>5</number>
             </property>
             <property name="rightMargin">
              <number>6</number>
             </property>
             <property name="bottomMargin">
              <number>5</number>
             </property>
             <item>
              <widget class="QToolButton" name="toolButton_logo1">
               <property name="minimumSize">
                <size>
                 <width>30</width>
                 <height>27</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>33</width>
                 <height>27</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">background-image: url(:/icon/resource/button.png);
font-size: 16px;</string>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="iconSize">
                <size>
                 <width>30</width>
                 <height>27</height>
                </size>
               </property>
               <property name="autoRaise">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QToolButton" name="toolButton_logo2">
               <property name="minimumSize">
                <size>
                 <width>30</width>
                 <height>27</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>33</width>
                 <height>27</height>
                </size>
               </property>
               <property name="layoutDirection">
                <enum>Qt::LeftToRight</enum>
               </property>
               <property name="styleSheet">
                <string notr="true">background-image: url(:/icon/resource/button.png);
font-size: 16px;</string>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="iconSize">
                <size>
                 <width>30</width>
                 <height>27</height>
                </size>
               </property>
               <property name="toolButtonStyle">
                <enum>Qt::ToolButtonIconOnly</enum>
               </property>
               <property name="autoRaise">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label">
               <property name="styleSheet">
                <string notr="true">font-size: 14px;
font-family: Adobe Heiti Std;
font-weight: normal;
color: #8E3122;</string>
               </property>
               <property name="text">
                <string>  作品展示</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QLabel" name="label_more">
               <property name="styleSheet">
                <string notr="true">font-size: 14px;
font-family: Adobe Heiti Std;
font-weight: normal;</string>
               </property>
               <property name="text">
                <string>查看更多</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QToolButton" name="toolButton">
               <property name="minimumSize">
                <size>
                 <width>16</width>
                 <height>16</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>15</width>
                 <height>16</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">background-image: url(:/icon/resource/more.png);</string>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="iconSize">
                <size>
                 <width>15</width>
                 <height>16</height>
                </size>
               </property>
               <property name="autoRaise">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="DListWidget" name="listWidget">
          <property name="styleSheet">
           <string notr="true">border-image: url(:/icon/resource/listItem.png);</string>
          </property>
          <property name="iconSize">
           <size>
            <width>30</width>
            <height>27</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DListWidget</class>
   <extends>QListWidget</extends>
   <header location="global">DListWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
